Clarify point reads only retrieve whole documents (#95835)
Azure/azure-content: articles/cosmos-db/sql/sql-query-getting-started.md

In Azure Cosmos DB SQL API accounts, there are two ways to read data:

> **Point reads** - You can do a key/value lookup on a single *item ID* and partition key. The *item ID* and partition key combination is the key and the item itself is the value. For a 1 KB document, point reads typically cost 1 [request unit](../request-units.md) with a latency under 10 ms. Point reads return a single item and can only return whole documents, not partial ones.

Reviewer: ```suggestion
**Point reads** - You can do a key/value lookup on a single *item ID* and partition key. The *item ID* and partition key combination is the key and the item itself is the value. For a 1 KB document, point reads typically cost 1 [request unit](../request-units.md) with a latency under 10 ms. Point reads return a single whole item, not a partial item or a specific field.
```

Changed the language to emphasize **item** instead of document. Otherwise, this looks acceptable. @jcocchi, any opinion on this?
